src/compute/src/render/sinks.rs

```diff
@@ -50,6 +51,7 @@ where
         start_signal: StartSignal,
         ct_times: Option<VecCollection<G, (), Diff>>,
         output_probe: &Handle<mz_repr::Timestamp>,
+        read_only_rx: watch::Receiver<bool>,
```

Comment:
You could also consider putting the `watch::Receiver` in the `CollectionState`, next to the `watch::Sender`. That would save you from having to plumb the extra argument through everywhere. In sink rendering you can get at the `CollectionState` using `compute_state.expect_collection(sink_id)`, there is some precedent for doing that already.